src/platform/intel/cavs/include/cavs/lib/pm_memory.h

```diff
@@ -84,6 +84,19 @@ static inline void cavs_pm_memory_hp_sram_mask_set(uint32_t mask, int segment,
 		if (delay >= MEMORY_POWER_CHANGE_TIMEOUT)
 			platform_panic(SOF_IPC_PANIC_MEM);
 	}
+
+#if CAVS_VERSION == CAVS_VERSION_2_5
```

Comment:
This code is recommended for all cAVS platforms, not only 2_5. as this is "cavs_pm_" function, this check needs to be removed.